app/instance-initializers/ember-a11y.js

```diff
@@ -1,10 +1,12 @@
 // This initializer exists only to make sure that the following
 // imports happen before the app boots.
 import Ember from 'ember';
-import { registerKeywords } from 'ember-a11y/ember-internals';
 import pivotRouteIdentifier from 'ember-a11y/utils/pivot-route-identifier';
 
+//{{htmlbars}}
+import { registerKeywords } from 'ember-a11y/ember-internals';
```

Comment:
Can we structure this like liquid-fire?  Basically, the `ember-a11y/ember-internals` module does the work directly, and you just get different versions of that module based on your Ember version.